src/components/SideMenu/sidemenu.import.less

```diff
@@ -0,0 +1,85 @@
+.sidemenu {
+  font-family: "Arial", serif;
+  background-color: #f4f4f4;
+  z-index: 99999999;
```

Comment:
@nicholasmiller1 This is way more z-index than you need. What if you have a modal you want to have above it? Then you have to type out 100,000,000. 

I like to keep everything under 10, then if I have something that needs to go on top of everything, I put it at 100. Keeps it clean that way, and I have a rough idea of what general plane things are on.